charter-template.html

```diff
@@ -204,6 +204,24 @@ <h3>
 
         </section>
 
+        <section id="incubation">
+          <!-- If no incubation or not a Working Group, delete the section. -->
+          <h3>
+            Tentative Deliverables
+          </h3>
+          <p>Depending on the incubation progress, interest from multiple implementers, and the consensus
+            of the Group participants, the Working Group may also produce Recommendation-track specifications
+            for the following documents:</p>
```

Comment:
The phrasing "may produce Rec-track specification for the following documents" is odd; you don't produce a spec FOR a document.  I think there are two distinct cases - when you're adopting an already-known incubation, in which case I think it should say (with the incubation/interest/consensus caveats) "may adopt the following documents as Rec-track specifications."   You could also (additionally), if you're trying to leave space for where the incubation comes from, simply say "may produce Rec-track specifications in the following areas", and change the "draft state" to "possible adoption inputs" or something like that.